jdbc-handler/src/main/java/org/apache/hive/storage/jdbc/dao/GenericJdbcDatabaseAccessor.java

```diff
@@ -228,6 +233,13 @@ protected Properties getConnectionPoolProperties(Configuration conf) {
       }
     }
 
+    // handle password
+    Credentials credentials = UserGroupInformation.getCurrentUser().getCredentials();
+    if (credentials.getSecretKey(DBCP_PWD) != null) {
+      LOGGER.info("found token in credentials");
```

Comment:
This was actually pretty useful in determining what the problem is on the cluster. It only logs once per query fragement. Ok to leave it?